packages/clay-charts/demos/index.html

```diff
@@ -463,6 +468,64 @@ <h2>Bubble</h2>
 			type: 'bubble'
 		}, '#bubble');
 
+		var predictiveChart = new metal.PredictiveChart({
+			axisXTickFormat: '%b',
+			data: [
+				{
+					data: [
+						'2018-01-01',
+						'2018-02-01',
+						'2018-03-01',
+						'2018-04-01',
+						'2018-05-01',
+						'2018-06-01',
+						'2018-07-01',
+						'2018-08-01',
+						'2018-09-01',
+						'2018-10-01',
+						'2018-11-01',
+						'2018-12-01'
+					],
+					id: 'x'
```

Comment:
Is this a convention for time series? Where does it say that this becomes the x-axis data?

Reply:
Yes, in billboard.js if a column has the id `x` it is used as the x-axis data.

Reply:
👍

Reply:
Isn't there a shortcut, or more meaningful API for it?

Reply:
I'll check that out right now, (I agree that it's not very meaningul)
Otherwise I was thinking we could maybe add a `timeseries` paremeter that would be used to  generated this column.

Reply:
Yeah, that sounds better...

Reply:
👍Ok, thanks